### ISSUE TITLE
Use Pretrained Inception-BatchNorm Network problem

### ISSUE THREAD

**Issue by ommiissyu** [dmlc/mxnet#1631]
hi, 

After loading the pretrained model, there are two ways to get the softmax layer's value. 

One method is

```
softmax_feature_1 = model.predict(batch)
```

The other method is
```
internals = model.symbol.get_internals()
softmax_symbol = internals["softmax_output"]
feature_extractor = mx.model.FeedForward(ctx=mx.gpu(), symbol=fea_symbol, numpy_batch_size=1,
                                         arg_params=model.arg_params, aux_params=model.aux_params,
                                         allow_extra_params=True)
softmax_feature_2 = feature_extractor.predict(batch)
```

Should softmax_feature_1's value and softmax_feature_2's value be the same? After printing the two values, I found the two values are different.
